node-packages/commons/src/tasks.ts

```diff
@@ -531,7 +536,7 @@ export const getControllerBuildData = async function(deployTarget: any, deployDa
   // if no build image is determined, the `remote-controller` defined default image will be used
   // once it reaches the remote cluster.
 
-  // @TODO: openshiftProject here can't be generated on the cluster side (it should be) but the addOrUpdate mutation doesn't allow for openshiftProject to be optional
+  // @TODO: envNamespaceName here can't be generated on the cluster side (it should be) but the addOrUpdate mutation doesn't allow for envNamespaceName to be optional
```

Comment:
`envNamespaceName` is not a variable, is this comment out of date? I'm not sure that the `@TODO` would be in this case, since the previous comment also doesn't apply anymore?

Reply:
Yah, that shouldn't have been there